README.md

```diff
@@ -59,6 +59,103 @@ style guide that you will not find in the GitHub one.
 
 ## Standards ##
 
+### Naming Convention ###
+
+The Apple Swift [API design guidelines](https://swift.org/documentation/api-design-guidelines/) promote clarity over brevity.
+
+Capitalize all type names with camel case, while function and variable names start with a lower case.
+
+```swift
+class MyClass { }
+protocol MyProtocol { }
+func myFunction { }
```

Comment:
This should be `func myFunction() { }`. Function definition requires parentheses.